XLS-0065d - Single Asset Vault (#128)
ripple/opensource.ripple.com: docs/xls-65d-single-asset-vault/reference/transactions/vault-deposit.md

| Field Name              | JSON Type     | Internal Type | Required? | Description         |
| :-----------------------| :------------ | :------------ | :-------- | :-------------------|
| `VaultID`               | String        | Hash256       | Yes       | The unique identifier of the vault to which the asset is deposited. |
| `Amount`                | String/Object | Amount        | Yes       | The asset and quantity to be deposited into the vault.|

The deposited asset must match the vault’s designated asset for the transaction to succeed. Depending on the asset type, the following changes occur:

- **XRP**: The vault’s `pseudo-account` balance increases, and the depositor’s balance decreases.
- **Fungible Token**: The [trust line](https://xrpl.org/docs/concepts/tokens/fungible-tokens#trust-lines) balance between the vault's `pseudo-account` and the asset issuer is adjusted.
- **MPT**: The `MPToken.MPTAmount` of both the depositor and the vault's `pseudo-account` is updated.

## {% $frontmatter.seo.title %} Flags

There are no flags defined for {% code-page-name /%} transactions.

## Transfer Fees

> A Single Asset Vault does not apply the [Transfer Fee](https://xrpl.org/docs/concepts/tokens/transfer-fees) to {% code-page-name /%} transactions. Additionally, whenever a protocol moves assets from or to a vault, the Transfer Fee must not be charged.

Reviewer: ```suggestion
A single asset vault does not apply the [transfer fee](https://xrpl.org/docs/concepts/tokens/transfer-fees) to {% code-page-name /%} transactions. Additionally, whenever a protocol moves assets from or to a vault, the transfer fee isn't charged.
```
